### PR TITLE
create daily habit

### LINKED ISSUE #5: Create DailyHabit entity
should have day(type Date) and checked(boolean)

### PR DESCRIPTION
**Pull request by chrislaynexavier**
fix #5

### CODE REVIEW

**Review by iurynogueira — APPROVED** [review]

File: node/test/DailyHabit.spec.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { DailyHabit } from "../src/domain/entity/DailyHabit"
+
+describe('daily Habit', () => {
+  let newDailyHabit!: DailyHabit
+  beforeEach(() => {
+    newDailyHabit = new DailyHabit(new Date())
+  })
+
+  it('should have day', () => {
```
Comment (edimossilva) [inline node/test/DailyHabit.spec.ts:9]: Hey this spec sounds  like testing the Date class. 
I think we should test if date is an instance of Date class. And do not test `getDate()` and others

Also, I would prefer to name the `it` as should have a date

File: node/test/DailyHabit.spec.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    expect(newDailyHabit.date.getDate()).toBe(new Date().getDate())
+    expect(newDailyHabit.date.getMonth()).toBe(new Date().getMonth())
+    expect(newDailyHabit.date.getFullYear()).toBe(new Date().getFullYear())
+  })
+
+  it('should habit start with check false', () => {
+    expect(newDailyHabit.checked).toBe(false)
+  })
+
+  it('should check the habit', () => {
```
Comment (edimossilva) [inline node/test/DailyHabit.spec.ts:19]:
```suggestion
  it('should have checked equals to true when toggle once', () => {
```

File: node/test/DailyHabit.spec.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    newDailyHabit.toggleCheck()
+    expect(newDailyHabit.checked).toBe(true)
+  } )
+
+  it('should be check false', () => {
```
Comment (edimossilva) [inline node/test/DailyHabit.spec.ts:24]: it('should have checked equals to false when toggle twice', () => {

**Review by edimossilva — APPROVED** [review]
Nice!
I left some  optional comments about patterns that I follow when writing specs

**Pull request merged**